widget/scroller.go

```diff
@@ -354,6 +354,24 @@ func (s *ScrollContainer) CreateRenderer() fyne.WidgetRenderer {
 	return scr
 }
 
+
+//ScrollToBottom will scroll content to container bottom - to show latest info which end user just added
+func (s *ScrollContainer) ScrollToBottom(isNeedRefresh bool) {
+	s.Offset.Y = s.Content.Size().Height - s.Size().Height + s.MinSize().Height
```

Comment:
Why is MinSize() needed here? The scroller Size() should be sufficient...
Y should range from 0 to (contentHeight - scrollerHeight)...

Reply:
Both ok.   Original "s.Content.Size().Height - s.Size().Height" already good, but the end would have some space.  And if I use "s.Content.Size().Height - s.Size().Height + s.MinSize().Height" it will scroll to the real end.

Reply:
The test `TestScrollContainer_Scrolled_Limit` shows that the furthest offset is content size - scroll size.
Adding MinSize should, in theory, make no difference, it will be clipped by `scroller.go:440`.
Do you have an example of how the gap is left when you don't adjust like you say?

Reply:
Done to change back to (s.Content.Size().Height - s.Size().Height)